src/cls/CMakeLists.txt

```diff
@@ -135,3 +135,17 @@ endif (WITH_CEPHFS)
 
 add_library(cls_numops_client STATIC numops/cls_numops_client.cc)
 
+add_library(cls_lua SHARED
+    lua/cls_lua.cc
+    lua/lua_bufferlist.cc
+)
+set_target_properties(cls_lua PROPERTIES VERSION "1.0.0" SOVERSION "1")
+install(TARGETS cls_lua DESTINATION ${cls_dir})
+target_link_libraries(cls_lua
+    liblua
```

Comment:
Hmm, I'll try this naming change, but I don't see anything in rpmbuild docs that suggest the naming causes the auto dependency. What I'm wondering is if since liblua is being built as a shared library, but not being installed, that the dependency between the object class and liblua is being recorded. with a vstart setup this doesn't show up. linking statically into the object class may work.